Plugins/NativeCamera/Source/Apple/NativeCameraImpl.mm

```diff
@@ -48,7 +137,14 @@ - (id)init:(std::shared_ptr<Babylon::Plugins::Camera::Impl::ImplData>)implData;
         CameraTextureDelegate* cameraTextureDelegate{};
         AVCaptureSession* avCaptureSession{};
         CVMetalTextureCacheRef textureCache{};
-        id <MTLTexture> textureBGRA{};
+        id <MTLTexture> textureRGBA{};
+        id<MTLRenderPipelineState> cameraPipelineState{};
+        id<MTLDevice> metalDevice{};
+        id<MTLCommandQueue> commandQueue{};
+        id<MTLCommandBuffer> currentCommandBuffer{};
+        size_t width{0};
+        size_t height{0};
```

Comment:
This should probably be kept in a Camera::Impl::CameraDimensions struct to match the Android side

Reply:
It's `private` on the Android side, so I don't know why do we need this? For consistency maybe?

Reply:
I don't think this is a needed change for correctness, but there are two main reasons I think it might be preferable to keep it in a struct.

1.) The primary reason is to make it slightly easier to address the line item in #1107 to handle resizing of the camera textures when shifting orientation.  By keeping it in a CameraDimensions object we can pass it up via const pointer to monitor changes or reference without having to push it into a struct and do some casting every time.
2.) Keeping the way each implementation tracks its state as consistent as possible makes sense to me.  I think we will eventually want to get rid of the redundancy with the split NativeCameraImpl.h, and move the actual shared API into the shared NativeCamera.h with platform specific implementation details only being implemented in their specific .mm/cpp files. In this case we may want to pull CameraDimensions into the public contract since it should be the same across platforms.